sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
-## 4.2.0-beta.4 (Unreleased)
+## 4.2.0 (2021-06-08)
+
+### New Features
+
+- Added support for the 7.2 version of the Key Vault service API.
+
+### Bug Fixes
+
+- Fixed a bug with `beginDeleteCertificate` and `beginRecoverDeletedCertificate` in which unknown service errors wouldn't bubble up properly to the end users.
+- Fixed an issue where importing a certificate incorrectly required a Subject or Subject Alternative Name.
+- Fixed an issue where retrying a failed initial Key Vault request may result in an empty body.
+
+### Changes since 4.2.0-beta.3
```

Comment:
Just checking - if I'm a "typical" user that avoided the previews and only used the GA/production quality packages do these breaks also apply to me? (and if so, maybe it's "Changes since 4.1.0") or similar.

Reply:
Yes, good call. I moved anything that is important from 4.1 -> 4.2 into the top level. Anything else is just from the previous beta.